src/sql/parse.py

```diff
@@ -53,37 +52,23 @@ def parse(cell, config):
         if len(pieces) == 1:
             return result
         cell = pieces[1]
-    # handle no space situation around =
-    if pieces[0].endswith("=<<"):
-        result["result_var"] = pieces[0][:-3]
```

Comment:
These logics were added as part of [this PR](https://github.com/ploomber/jupysql/pull/609/files). 
Any reason these are removed?

Reply:
Yes! the previous code focused on spacing around "=" which made sense for equal operator but since we are also focusing on "<<" spacing I could have added a similar logic around "<<"  and it would have worked but  increased the code size plus reduce clarity, my codes main idea is to keep "<<" pivot point and make code readable